packages/eslint-plugin-ckeditor5-rules/tests/index.js

```diff
@@ -7,10 +7,14 @@
 
 const fs = require( 'fs' );
 
+console.log( '\n🔷 eslint-plugin-ckeditor5-rules - running tests.\n' );
+
 const tests = fs.readdirSync( __dirname ).filter( test => {
 	return test !== 'index.js';
 } );
 
 for ( const test of tests ) {
 	require( `./${ test }` );
 }
+
+console.log( '✅ eslint-plugin-ckeditor5-rules - tests passed with no errors.\n' );
```

Comment:
You don't know whether they pass. I'd say that execution of these tests is completed (without predicting a status).